### ISSUE TITLE
Consider moving deployhl72fhir.bash to using ARM or real infra deployment

### ISSUE THREAD

**Issue by mattpasco-microsoft** [microsoft/OpenHack-FHIR#23]
Using the `az cli` causes deployment failures as the resources are not fully provisioned, causing subsequent commands to fail. The process of prompting the user for 12 variables is tedious.

**Comment by cyberuna** [comment]
Agree, please post this in the original Github: https://github.com/microsoft/health-architectures

**Comment by mattpasco-microsoft** [comment]
tracking here: https://github.com/microsoft/health-architectures/issues/23
